pkg/process/checks/process_nix.go

```diff
@@ -76,5 +77,9 @@ func calculatePct(deltaProc, deltaTime, numCPU float64) float32 {
 	}
 
 	// In order to emulate top we multiply utilization by # of CPUs so a busy loop would be 100%.
-	return float32(overalPct * numCPU)
+	pct := overalPct * numCPU
```

Comment:
Do you mind summarizing the "motivation" section here? It would be nice to have this captured in code